apps/cookbook/src/app/showcase/item-showcase/item-showcase.component.html

```diff
@@ -211,6 +294,18 @@ <h3>Numeric</h3>
   <cookbook-item-example-input-numeric #inputNumericExample></cookbook-item-example-input-numeric>
 </cookbook-example-viewer>
 
+<h2 #disclosure>Disclosure slot</h2>
+<cookbook-example-viewer [html]="disclosureExample.template">
+  <cookbook-item-example-disclosure #disclosureExample></cookbook-item-example-disclosure>
+</cookbook-example-viewer>
+<p class="comment small">
+  <em
+    ><strong>Please note</strong>, set disclosure attribute on <code>kirby-item</code> to current
+    available icons in disclosure slot
+    <code> 'link' | 'arrow-more' | 'arrow-down' | 'arrow-up'</code>
+  </em>
+</p>
```

Comment:
```suggestion
    ><strong>Please note</strong>, the disclosure input is limited to a set of predefined icons: <code> 'link' | 'arrow-more' | 'arrow-down' | 'arrow-up'</code>
  </em>
</p>
```

Consider something like this instead.

Reply:
Suggestion taken!